### DISCUSSION TITLE
Bora Binary Explorer V1 with Logisim Evolution?

### DISCUSSION THREAD

**Discussion by Greydesk** [logisim-evolution/logisim-evolution#1998] [Q&A] [answered]
Hi all,
I have a version 1 Bora Binary Explorer that I received when I was doing my EE degree (https://hackaday.com/2012/10/29/bora-board-teaches-binary-hardware/). It uses the older Xilinx software (version 10 I believe) and a XC9536XL CPLD as its main unit. I was hoping that I could start using it again but using Logisim-Evolution as I run Linux as my daily driver and I've not had good success with the Xilinx software in Linux. It runs but there are issues compiling and uploading to the board. 
Has there been any work already done with this board, this CPLD, or can someone lend a hand getting it to work? I don't mind doing all the legword and posting the logs, I just don't know where to get started with it.
Thanks!

**Comment by maehne** [comment, marked as answer]
If I remember right, the XC9536XL CPLD was still supported in Xilinx ISE 14.7. This toolchain is not maintained by Xilinx since many years and it has become increasingly difficult to make it work on modern operating systems. Nevertheless, if you got that toolchain working and the CPLD recognised by its programming tool, you should be able to add support for your particular board to Logisim-evolution using [these instructions](https://github.com/logisim-evolution/logisim-evolution/issues/376).
